pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/OperatorUtils.java

```diff
@@ -51,6 +52,7 @@ private OperatorUtils() {
    * @return Canonicalize form of the input function name
    */
   public static String canonicalizeFunctionName(String functionName) {
+    functionName = StringUtils.replace(functionName, " ", "_");
```

Comment:
We should just remove the space. See `TransformFunctionFactory.canonicalize()` for example